README.md

```diff
@@ -1,6 +1,7 @@
-Pandamonium
-===========
+# Pandamonium
```

Comment:
why you always gotta hate on my underline style? I like the thing that looks more like pretty in raw text form.

Reply:
You're limited in the level of headings that you can use with this RST-like format. Additionally, it looks very much like the document is mixed RST and Makrdown which is pretty strange. (In all seriousness, have you tried writing in RST before? I'm not sure if you would like it more or less than Markdown).

I'll change this back though.

Reply:
@dguest Is it possible to go to the equivalent of `###` heading with this RST-like syntax? I didn't rewrite all of the README with this as it isn't clear how I'd do so given this limitation. It seems that this only goes to h2 headers given https://www.markdownguide.org/basic-syntax/#headings

Reply:
yeah that's true. I always thought it was fine because `###` it emphasizes these points less than a big fat underline. But I shouldn't hold up a MR over this obviously!